samples/payment/CreateDepositPayment.js

```diff
@@ -10,7 +10,6 @@ const request = {
   price: 100.0,
   buyerMemberId: 1, // change it with a valid wallet owner, aka member id
   conversationId: '456d1297-908e-4bd6-a13b-4be31a6e47d5',
-  currency: Craftgate.Model.Currency.TRY,
```

Comment:
Kaldırmayalım derim bunu. Zaten kontrolümüz var.

Reply:
Request'te artık zaten yer almadığı için kaldırmamız uygun